ganttproject/src/net/sourceforge/ganttproject/GanttProject.java

```diff
@@ -902,7 +902,7 @@ public static boolean main(String[] arg) {
       return false;
     }
     if (mainArgs.log && "auto".equals(mainArgs.logFile)) {
-      mainArgs.logFile = System.getProperty("user.home") + File.separator + "ganttproject.log";
+      mainArgs.logFile = System.getProperty("user.home")+ File.separator + ".ganttproject.d" + File.separator + "ganttproject.log";
```

Comment:
This is likely to fail if directory doesn't exist, but even if it is fixed or doesn't fail, I don't think that we should create hidden directory `.ganttproject.d` on all systems.

When talking to Windows/Mac OSX users it is way more easier to tell them _find `ganttproject.log` file in their home directory_ than _find a hidden directory `.ganttproject.d` (you may need to enable hidden files showing in your file manager), then find `ganttproject.log` file in that directory_

Besides, issue #1309  requests chaning log directory to `/var/log/` or whatever user specifies in the command line, rather than to hidden directory.